Misc/NEWS.d/next/Core_and_Builtins/2025-01-10-18-56-20.gh-issue-128563.baDvls.rst

```diff
@@ -0,0 +1 @@
+A new type of interpreter has been added to CPython. This interpreter uses tail calls for its instruction handlers. Preliminary benchmark results suggest 7.4-14.7% geometric mean faster on pyperformance (depending on platform), and up to 45% faster on Python-intensive workloads. Patch by Ken Jin, with ideas by Garret Gu, Haoran Xu, and Josh Haberman.
```

Comment:
```suggestion
A new type of interpreter has been added to CPython. This interpreter uses tail calls for its instruction handlers. Benchmark results show a 7-14% geometric mean faster on pyperformance (depending on platform), and up to 45% faster on Python-intensive workloads. Patch by Ken Jin, with ideas by Garret Gu, Haoran Xu, and Josh Haberman.
```
(number can always be updated if needed, but the news entry should not state preliminary)
(I would round to whole number, the accuracy is not that fine of the benchmarks)